### PR TITLE
Remove envelope from template strings

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This PR removes `.Envelope` from the template to resolve an issue preventing data from displaying on the transaction accept preview page.

Note: The form should display the latest secure envelope info. We currently aren't able to create a secure envelope in development, so this hasn't been completely tested yet.

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/templates/partials/transaction/transaction_accept.html [hunk 1]
```diff
@@ -1,39 +1,38 @@
 <section class="my-4">
-  {{ if .Envelope }}
   <section>
     <div class="mb-4 md:mb-0">
-      <span class="font-semibold">Transaction ID: {{ .Envelope.Transaction.Txid }}</span>
+      <span class="font-semibold">Transaction ID: {{ .Transaction.Txid }}</span>
```
Comment (daniellemaxwell) [inline pkg/web/templates/partials/transaction/transaction_accept.html:4]: The `Txid` is included in the response, but we may need the `TransactionID` here.

Comment (pdeziel) [reply pkg/web/templates/partials/transaction/transaction_accept.html:4]: Can you remind me what the difference is between those two?

**Review by pdeziel — APPROVED** [review]
Looks good, I just had one question mostly for my curiosity.

File: pkg/web/templates/partials/transaction/transaction_accept.html (hunk 1 above)
Comment (daniellemaxwell) [reply pkg/web/templates/partials/transaction/transaction_accept.html:4]: The `transaction ID` is generated by the originator and it must be a UUID. The `Txid` is an ID that's unique to the blockchain or network.

**Pull request merged**